cardano-node/src/Cardano/Node/Configuration/POM.hs

```diff
@@ -188,8 +199,22 @@ instance FromJSON PartialNodeConfiguration where
            , pncValidateDB = mempty
            , pncShutdownIPC = mempty
            , pncShutdownOnSlotSynced = mempty
+           , pncMaybeMempoolCapacityOverride
            }
     where
+      parseMempoolCapacityBytesOverride :: Object -> J.Parser (Maybe MempoolCapacityBytesOverride)
```

Comment:
The convention for qualifying imports for `Data.Aeson` is `as Aeson`. Lets stick to that to keep the importing styles uniform.